cogs5e/lookup.py

```diff
@@ -244,7 +244,8 @@ def __init__(self, bot):
                     level_resources[table['colLabels'][i]] = parse_data_entry([col])
 
             for res_name, res_value in level_resources.items():
-                embed.add_field(name=res_name, value=res_value)
+                if res_value != '0':
```

Comment:
As far as I can tell this only happens with higher level spell slots and a monk's Ki at level 1, so this should be fine.